internal/transport/controlbuf.go

```diff
@@ -191,7 +191,7 @@ type goAway struct {
 	code      http2.ErrCode
 	debugData []byte
 	headsUp   bool
-	closeConn bool
+	closeConn error
```

Comment:
Document since it's no longer as obvious: `// if set, loopyWriter will exit, resulting in conn closure`

Reply:
Done.